igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -86,10 +84,14 @@ class TravisBuildMonitor implements PollingMonitor{
     @Value('${travis.repositorySyncEnabled:false}')
     Boolean repositorySyncEnabled
 
+    @SuppressWarnings('GStringExpressionWithinString')
+    @Value('${travis.cachedJobTTLDays:60}')
```

Comment:
I just put a value there, we are setting it to 7 days here. I guess it depends on how many jobs you have.